projects/common/src/time/page-time-range-preference.service.ts

```diff
@@ -32,11 +33,19 @@ export class PageTimeRangePreferenceService {
   public getTimeRangePreferenceForPage(rootLevelPath: string): Observable<TimeRangeResolver> {
     return combineLatest([
       this.pageTimeRangeStringDictionary$,
-      this.featureStateResolver.getFeatureState(ApplicationFeature.PageTimeRange)
+      this.featureStateResolver.getFeatureState(ApplicationFeature.PageTimeRange),
+      this.featureStateResolver.getFeatureFlagValue(ApplicationFeature.TenantDefaultTimeRangeMap)
     ]).pipe(
-      map(([pageTimeRangeStringDictionary, featureState]) => {
+      map(([pageTimeRangeStringDictionary, featureState, tenantDefaultTimeRangeMap]) => {
         if (featureState === FeatureState.Enabled) {
           if (isNil(pageTimeRangeStringDictionary[rootLevelPath])) {
+            if (this.tenantTimeRangeMapHasValue(tenantDefaultTimeRangeMap, rootLevelPath)) {
```

Comment:
nit: flip around the conditions to avoid nesting:
if has pref override, return
if has map override, return
else return default for path